ios/brave-ios/Sources/Brave/Frontend/Shields/ShieldsPanelView.swift

```diff
@@ -0,0 +1,413 @@
+          ) {
+            actionCallback(.shredSiteData)
+          }
+        } label: {
+          ShieldSettingsNavigationWrapper {
+            Text(Strings.Shields.shredSitesData)
+              .frame(maxWidth: .infinity, alignment: .leading)
+              .multilineTextAlignment(.leading)
+          }
+        }
+        .foregroundStyle(Color(.bravePrimary))
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .padding(.vertical, 4)
+      }
+    }
+  }
+
+  @ViewBuilder private var globalSettingsSectionView: some View {
+    ShieldSettingSectionHeader(title: Strings.Shields.globalControls)
+    ShieldSettingRow {
+      Button {
+        actionCallback(.navigate(.globalShields, dismiss: true))
+      } label: {
+        ShieldSettingsNavigationWrapper {
+          Label(
+            Strings.Shields.globalChangeButton,
+            braveSystemImage: "leo.globe.block"
+          )
+          .frame(maxWidth: .infinity, alignment: .leading)
+          .multilineTextAlignment(.leading)
+          .foregroundStyle(Color(.bravePrimary))
+          .labelStyle(.titleAndIcon)
+        }
+      }
+      .buttonStyle(.plain)
+      .frame(maxWidth: .infinity, alignment: .leading)
+    }
+  }
+
+  /// Load the favicon from a site URL directly into a `Image`
+  private func loadFavicon(
+    for siteURL: URL,
+    isPrivateBrowsing: Bool
+  ) async {
+    do {
+      guard
+        let faviconImage = try await FaviconFetcher.loadIcon(
+          url: siteURL,
+          persistent: !isPrivateBrowsing
+        ).image
+      else { return }
+      self.favicon = Image(uiImage: faviconImage)
+    } catch {
+      ContentBlockerManager.log.error(
+        "Failed to load favicon: \(String(describing: error))"
+      )
+    }
+  }
+}
+
+private struct ShieldSettingRow<Contents>: View where Contents: View {
+  @ViewBuilder var contents: () -> Contents
+
+  var body: some View {
+    VStack(alignment: .leading, spacing: 0) {
+      contents()
+        .frame(minHeight: 32, alignment: .center)
+        .padding(.horizontal)
+        .padding(.vertical, 4)
+      Divider()
+    }
+    .frame(maxWidth: .infinity, alignment: .leading)
+  }
+}
+
+private struct ShieldSettingsNavigationWrapper<Contents>: View where Contents: View {
+  @ViewBuilder var contents: () -> Contents
+
+  var body: some View {
+    HStack {
+      contents()
+      // Hack to showing the navigation chevron
+      Image(systemName: "chevron.right")
+        .font(.footnote)
+        .fontWeight(.medium)
+        .foregroundStyle(Color(.secondaryBraveLabel))
+    }.contentShape(Rectangle())
+  }
+}
+
+private struct ShieldSettingSectionHeader: View {
+  let title: String
+
+  var body: some View {
+    VStack(alignment: .leading, spacing: 8) {
+      Text(title)
+        .font(.footnote)
+        .foregroundStyle(Color(.secondaryBraveLabel))
+        .textCase(.uppercase)
+        .padding(.horizontal)
+        .padding(.top, 8)
+        .padding(.bottom, 0)
```

Comment:
Why do you need padding of zero? Is this line necessary?

Reply:
Just so there is no space between the advanced header and bottom when advanced section is collapsed as was the original design.